modules/products/products.service.js

```diff
@@ -0,0 +1,77 @@
+const Products = require('./products.model');
+const Size = require('../../models/Size');
+
+class ProductsService {
+  getAllProducts() {
```

Comment:
Please, merge `getProductsByOptions` and `getAllProducts` into one method that will take filter but if the filter is not provided will use the default one

Reply:
done